.github/actions/rust-setup/action.yaml

```diff
@@ -10,3 +10,6 @@ runs:
     - uses: bmwill/rust-cache@v1
       with:
         path: ~/.cargo/registry/src/**/librocksdb-sys-*
+    - name: install Prover dependencies
+      shell: bash
+      run: scripts/dev_setup.sh -b -p -y
```

Comment:
what are the flags for? prefer using long form of the flags or add a comment on what's the intention